### PR TITLE
ARP/Update metadata: OpenSight.FlashFXP version 5.4.0.3970

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21575)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
image
This might cause issues
image

File: manifests/o/OpenSight/FlashFXP/5.4.0.3970/OpenSight.FlashFXP.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://get.flashfxp.com/ftp/client/download/FlashFXP54_3970_Setup.exe
+  InstallerSha256: 631AE7074649A665D62AC6FC940D203EFF715C88B4B57EE46865286607909231
+  InstallerSwitches:
+    Silent: /s
```
Comment (OfficialEsco) [inline manifests/o/OpenSight/FlashFXP/5.4.0.3970/OpenSight.FlashFXP.installer.yaml:18]:
```suggestion
- silent
- silentWithProgress
Installers:
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: exe
  Scope: machine
  InstallerUrl: https://get.flashfxp.com/ftp/client/download/FlashFXP54_3970_Setup.exe
  InstallerSha256: 631AE7074649A665D62AC6FC940D203EFF715C88B4B57EE46865286607909231
  InstallerSwitches:
    Silent: /S
    SilentWithProgress: /QUIET
    InstallLocation: TARGETDIR=<INSTALLPATH>
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Maybe like this?
Seems like it doesn't search for update when doing Silent/Quiet
PERUSER dosn't seem like a feature we can use per now because it requires administrator to start the installer
RESTARTAPP=TRUE might be nice?

File: manifests/o/OpenSight/FlashFXP/5.4.0.3970/OpenSight.FlashFXP.installer.yaml (hunk 1 above)
Comment (quhxl) [reply manifests/o/OpenSight/FlashFXP/5.4.0.3970/OpenSight.FlashFXP.installer.yaml:18]: LGTM. Let us try this. Thanks!

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**